envoy/type/BUILD

```diff
@@ -0,0 +1,14 @@
+load("//bazel:api_build_system.bzl", "api_proto_library", "api_go_proto_library")
+
+licenses(["notice"])  # Apache 2
+
+api_proto_library(
+    name = "range",
+    srcs = ["range.proto"],
+    visibility = ["//visibility:public"],
```

Comment:
I think this can just be elided and it will work by magic with the default.

Reply:
@htuch Removing the visibility line or passing empty does not work.

The default in https://github.com/envoyproxy/data-plane-api/blob/master/bazel/api_build_system.bzl#L81 
is private.

ERROR: /source/envoy/api/v2/route/BUILD:5:1: Target '//envoy/type/v2:range' is not visible from target '//envoy/api/v2/route:route'. Check the visibility declaration of the former target if you think the dependency is legitimate